### DISCUSSION TITLE
Login stops working shortly after initial self-host setup

### DISCUSSION THREAD

**Discussion by damienbutt** [Infisical/infisical#1841] [Q&A]
Hi all,

I've been trying to get up and running with Infisical but I am having trouble.

During the initial super admin creation, it throws an error saying "admin not created" or something to that effect. But then when you try again, it says, "admin account already exists" (or something similar). I can log in and start configuring the service. But upon logout, I can no longer log in again. It just says the password is invalid.

Any ideas where I could be going wrong?

Thanks,
D

**Comment by AdityaGoyal1999** [comment]
This is not an answer but I was experiencing the same issue. This seems to be an issue
